libc/config/baremetal/arm/entrypoints.txt

```diff
@@ -247,6 +247,8 @@ set(TARGET_LIBM_ENTRYPOINTS
     libc.src.math.cos
     libc.src.math.cosf
     libc.src.math.coshf
+    libc.src.math.daddl
```

Comment:
Don't enable entrypoints for baremetal yet, since we cannot test them downstream, and we don't want to revert the whole PR if downstream baremetal bots fail.